src/serve_data.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
-const fs = require('fs');
-const path = require('path');
-const zlib = require('zlib');
+import fs from 'node:fs';
+import path from 'path';
+import zlib from 'zlib';
 
-const clone = require('clone');
-const express = require('express');
-const MBTiles = require('@mapbox/mbtiles');
-const Pbf = require('pbf');
-const VectorTile = require('@mapbox/vector-tile').VectorTile;
+import clone from 'clone';
+import express from 'express';
+import MBTiles from '@mapbox/mbtiles';
+import Pbf from 'pbf';
+import VectorTile from '@mapbox/vector-tile';
 
-const utils = require('./utils');
+import {getTileUrls, fixTileJSONCenter} from './utils.js';
 
-module.exports = {
+export const serve_data = {
```

Comment:
Maybe also not for this PR, but we might consider replacing all of the snake_case exports with camelCase ones.